modules/core/tests/client/helpers/generate-fake-data.js

```diff
@@ -0,0 +1,104 @@
+import faker from 'faker';
+import sample from 'lodash/sample';
+import sampleSize from 'lodash/sampleSize';
```

Comment:
Probably no difference in tests if you group `{ sample, sampleSize } from 'lodash'` or not since we don't need to worry about bundle size here. Up to you.

Reply:
Makes sense. :+1:

2b2dde6c1b41b6722f797191c9f97fa6b0069124